### ISSUE TITLE
Incorrect `repository` property in `package.json`

### ISSUE THREAD

**Issue by silverwind** [citation-js/plugin-software-formats#14]
[This URL format](https://github.com/citation-js/plugin-software-formats/blob/a0868e0a22d3055a11caa4a033b3c66cd977e48d/packages/plugin-software-formats/package.json#L8) is invalid:

````json
"repository": "https://github.com/citation-js/plugin-software-formats/packages/plugin-software-formats",
````

The link itself is 404 and the [correct way](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository) to specify a subdirectory would be:

```json
"repository": {
  "type": "git",
  "url": "https://github.com/citation-js/plugin-software-formats.git",
  "directory": "packages/plugin-software-formats"
}
```

It seems the issue is present in all `citation-js` packages.

**Comment by larsgw** [comment]
Thank you for the report! When I converted this to a monorepo a few days ago I checked with https://github.com/citation-js/citation-js but that is indeed also wrong.

**Comment by larsgw** [comment]
Fixed in 6fb275b5ba3fa449ae1c3bc1af973cf2df076eff, it'll be included in the next release.
